src/components/molecules/Visualizer/Plugin/hooks.ts

```diff
@@ -86,9 +86,9 @@ export default function ({
 
   useEffect(() => {
     const visible = shownPluginModalInfo?.id === (widget?.id ?? block?.id);
-    if (modalVisible.current !== visible) {
-      modalVisible.current = visible;
-      if (!modalVisible.current) {
+    if (modalVisible !== visible) {
+      setModalVisibility(visible);
+      if (!visible) {
```

Comment:
Shouldn't this be modalVisible, not visible?

Reply:
Now `modalVisible` is a state and after the setState the value will not got updated immediately, so may use either `modalVisible` or `!visible` . I think `!visible` is more meaningful?